resources/README.md

```diff
@@ -23,3 +23,23 @@ path-to-referenced-charts:
 
 The version of the actual component image is located under the **global.{name_of_component}.version** property.
 **{name_of_component}** is a directory name of the component where dashes are replaced by underscores.
+
+### Add monitoring to components
+
+Include configuration files for Service Monitors, alert rules, and dashboards under your component's chart to ensure proper health check monitoring of your component. 
+
+For an example of such a component, see [Service Catalog](https://github.com/kyma-project/kyma/blob/master/resources/service-catalog/charts/catalog/templates) that contains the [ServiceMonitor](https://github.com/kyma-project/kyma/blob/master/resources/service-catalog/charts/catalog/templates/controller-manager-service-monitor.yaml) and [dashboard](https://github.com/kyma-project/kyma/blob/master/resources/service-catalog/charts/catalog/templates/dashboard-configmap.yaml) configurations.
+
+
+When creating a ServiceMonitor resource, follow this naming convention:
+
+| Resource | Description | Name/Pattern | Example |
```

Comment:
I would change the order of columns into: `| Resource |  Name/Pattern | Example | Description |`